<h2>Description</h2>
We describe a computational approach that integrates GRO-seq and RNA-seq data to annotate long noncoding RNAs lncRNAs, with increased sensitivity for low-abundance lncRNAs. 
We used this approach to characterize the lncRNA transcriptome in MCF-7 human breast cancer cells, including more than 700 previously unannotated lncRNAs. 
We then used information about the  transcription of lncRNA genes from GRO-seq, steady-state levels of lncRNA transcripts in cell lines and patient samples from RNA-seq, and  histone modifications and factor binding at lncRNA gene promoters from ChIP-seq to explore lncRNA gene structure and regulation, as well as lncRNA transcript stability, regulation, and function. 
Functional analysis of selected lncRNAs with altered expression in breast cancers revealed roles in cell proliferation, regulation of an E2F-dependent cell-cycle gene expression program, and estrogen-dependent mitogenic growth. 
Collectively, our studies demonstrate the use of an integrated genomic and molecular approach to identify and characterize growth-regulating lncRNAs in cancers.

<p>
This track represents the Bancer lncRNA track.
</p>


<h2>Methods</h2>
<p>
<ul>
<h3>Process 1</h3>
Kraus Lab Breast Bancer lncRNA track (pmid: 26236012)

Step1: Download gtfToGenePred and then genePredToBigGenePred For Linux: wget http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/gtfToGenePred wget http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/genePredToBigGenePred For MacOS: wget http://hgdownload.soe.ucsc.edu/admin/exe/macOSX.x86_64/gtfToGenePred wget http://hgdownload.soe.ucsc.edu/admin/exe/macOSX.x86_64/genePredToBigGenePred

wget http://genome.ucsc.edu/goldenPath/help/examples/bigGenePred.as wget http://hgdownload.soe.ucsc.edu/goldenPath/hg19/bigZips/hg19.chrom.sizes -O chrom.sizes wget http://hgdownload.soe.ucsc.edu/admin/exe/macOSX.x86_64/bedToBigBed

Step2: gtfToGenePred GSE63189_Catalog_of_lncRNAs_in_MCF-7cells.gtf GSE63189_Catalog_of_lncRNAs_in_MCF-7cells.genePred genePredToBigGenePred GSE63189_Catalog_of_lncRNAs_in_MCF-7cells.genePred GSE63189_Catalog_of_lncRNAs_in_MCF-7cells.bigGenePred #NOTE: The .bigGenePred file generated above does not have all the required columns.(https://genome.ucsc.edu/goldenpath/help/bigGenePred.html), and then convert it to a bigBed file.

Step3: cat GSE63189_Catalog_of_lncRNAs_in_MCF-7cells.bigGenePred | awk '{print $1,$2,$3,$4,$5,$6,$7,$8,$9,$10,$11,$12,$4,$13,$14,$15,$4,$4,$4,$16}' | sort -k1,1 -k2,2n > GSE63189_Catalog_of_lncRNAs_in_MCF-7cells.bigGenePred.txt

Step4: bedToBigBed -as=bigGenePred.as -type=bed12+8 GSE63189_Catalog_of_lncRNAs_in_MCF-7cells.bigGenePred.txt chrom.sizes GSE63189_Catalog_of_lncRNAs_in_MCF-7cells.bigGenePred.bb

Step5: Use the "Raw" text display for the link to hub to give the following URL https://raw.githubusercontent.com/Kraus-Lab/publichubs/master/breast_cancer_lncrna_pmid_26236012/hub.txt

<h3>Notes</h3>
For a full or a description of the methods used, refer to Sun <em>et al</em>. (2015) in the References section below.
</p>
</ul>

<h2>Credits</h2>
<p>
Data were generated and processed at the Laboratory of Signaling and Gene Regulation, Cecil H. and Ida Green Center for Reproductive Biology Sciences and Division of Basic Reproductive Biology Research, Department of Obstetrics and Gynecology, University of Texas Southwestern Medical Center, Dallas, TX 75390, USA. 
For inquiries, please contact us at the following address: lee.kraus(at) utsouthwestern.edu
</p>

<h2>References</h2>
<p>
Sun M, Gadad SS, Kim DS, Kraus WL <em>et al</em>.
<a href="https://pubmed.ncbi.nlm.nih.gov/26236012/">
Ensembl regulation resources</a>.
<em>Database (Mol Cell)</em>. 2015;2015.
PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/26236012" target="_blank">26236012</a>; PMC: <a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4546522/" target="_blank">PMC4546522</a>
</p>


